mne/io/brainvision/brainvision.py

```diff
@@ -469,7 +469,10 @@ def _get_eeg_info(vhdr_fname, eog, misc):
         else:
             name, _, resolution, unit = props[:4]
         ch_names[n - 1] = name
-        cals[n - 1] = float(resolution)
+        if len(resolution) > 0:
```

Comment:
It's a str (the result of splitting a string); ideally, it's a string that can be converted to a float, but in the case of my truncated (EEGLAB-exported) files, it's an empty string.